lib/components/intent.js

```diff
@@ -1,13 +1,16 @@
 "use strict";
-var Promise = require("bluebird");
-var MatrixUser = require("../models/users/matrix");
-var MatrixEvent = require("matrix-js-sdk").MatrixEvent;
-var RoomMember = require("matrix-js-sdk").RoomMember;
+const Promise = require("bluebird");
+const MatrixUser = require("../models/users/matrix");
+const MatrixEvent = require("matrix-js-sdk").MatrixEvent;
+const RoomMember = require("matrix-js-sdk").RoomMember;
+const ClientRequestCache = require("./client-request-cache");
 
-var STATE_EVENT_TYPES = [
+const STATE_EVENT_TYPES = [
     "m.room.name", "m.room.topic", "m.room.power_levels", "m.room.member",
     "m.room.join_rules", "m.room.history_visibility"
 ];
+const DEFAULT_CACHE_TTL = 30000;
```

Comment:
I think we can safely up this to a minute or two.

Reply:
I think 90 seconds is a pretty standard one?